datasets/superb/superb.py

```diff
@@ -148,6 +148,27 @@ class Superb(datasets.GeneratorBasedBuilder):
             data_url="http://www.openslr.org/resources/12/",
             task_templates=[AutomaticSpeechRecognition(audio_file_path_column="file", transcription_column="text")],
         ),
+        SuperbConfig(
+            name="ks",
+            description=textwrap.dedent(
+                """\
+            Keyword Spotting (KS) detects preregistered keywords by classifying utterances into a predefined set of
+            words. The task is usually performed on-device for the fast response time. Thus, accuracy, model size, and
+            inference time are all crucial. SUPERB uses the widely used [Speech Commands dataset v1.0] for the task.
+            The dataset consists of ten classes of keywords, a class for silence, and an unknown class to include the
+            false positive. The evaluation metric is accuracy (ACC)"""
+            ),
+            features=datasets.Features(
+                {
+                    "file": datasets.Value("string"),
+                    "label": datasets.Value("string"),
```

Comment:
For the "label" field, we should better use `datasets.ClassLabel` (similarly as tfds does).